packages/react-components/src/components/table-output-component.tsx

```diff
@@ -508,13 +516,28 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
         return lines[0].index;
     }
 
-    private async findNextMatchedEvent() {
+    private async findMatchedEvent(direction: Direction) {
         let isFound = false;
         if (this.gridApi) {
             this.gridApi.deselectAll();
+            let rowNodes: RowNode[] = [];
+            let currRowIndex = this.selectStartIndex === -1 ? this.gridApi?.getLastDisplayedRow() + 1 : this.selectStartIndex;
+            const blockSize = this.props.cacheBlockSize ? this.props.cacheBlockSize : 200;
```

Comment:
Just checking, will the 'current block' fix work correctly if the cachBlockSize is undefined and we use 200?

Reply:
I just chose 200 because that's the current blocksize. I don't think there is a "good" value if the cacheBlockSize is undefined.

Reply:
To clarify, what I'm worried is if we accurately compute the 'block number' for comparison in that case.